buildSrc/src/main/groovy/com.indeed.java-conventions.gradle

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
There is no reason for this to exist, and I would HIGHLY recommend against using buildSrc.

Simply put this all inside the root build.gradle inside an `allprojects { }` block. However, basically none of this should be used anyways, since all of this is included in the indeed com.indeed.oss plugin. The only thing that should even possibly be kept from here would be the dependencies block, but even that is a bit strange to just blindly include in every subproject.

Reply:
Removed